packages/transport-native/trezor-bridge-native.podspec

```diff
@@ -0,0 +1,19 @@
+require "json"
```

Comment:
rename to trezor-transport-native.podspec?

Reply:
renamed